src/core/dbus-util.h

```diff
@@ -250,7 +250,12 @@ static inline int bus_set_transient_usec(Unit *u, const char *name, usec_t *p, s
 static inline int bus_set_transient_usec_fix_0(Unit *u, const char *name, usec_t *p, sd_bus_message *message, UnitWriteFlags flags, sd_bus_error *error) {
         return bus_set_transient_usec_internal(u, name, p, true, message, flags, error);
 }
-int bus_verify_manage_units_async_full(Unit *u, const char *verb, const char *polkit_message, sd_bus_message *call, sd_bus_error *error);
+
+int bus_verify_manage_units_async_impl(Manager *manager, const char *id, const char *verb, const char *polkit_message, sd_bus_message *call, sd_bus_error *error);
+static inline int bus_verify_manage_units_async_full(Unit *u, const char *verb, const char *polkit_message, sd_bus_message *call, sd_bus_error *error) {
```

Comment:
it's a bit strange that we have bus_verify_manage_units_async_full(), and bus_verify_manage_units_async_impl() here, but bus_verify_manage_units() is defined elsewhere, and entirely unrelated?

I think these really should be cleaned up if this is touched, i.e. move to the same place and be wrappers of each other?